test_files/type/type.js

```diff
@@ -96,7 +94,7 @@ val => val + 1));
  */
 function typeGenericCallback(callback) { }
 typeGenericCallback((/**
- * @param {?} val
- * @return {?}
+ * @param {*} val
```

Comment:
It would be nice to be intentional about this. If think that `?` was better add a TODO. I think `*` is fine to force an explicit cast.